packages/block-editor/src/hooks/use-zoom-out.js

```diff
@@ -21,23 +21,50 @@ export function useZoomOut( zoomOut = true ) {
 	);
 	const { isZoomOut } = unlock( useSelect( blockEditorStore ) );
 
-	useEffect( () => {
-		const isZoomOutOnMount = isZoomOut();
+	const toggleZoomOnUnmount = useRef( null );
```

Comment:
Should this not start as null, instead default to true or false?

Reply:
Yeah, I considered starting it as false. I wasn't sure if I'd need to track if it had ever been set or not. If the code gets merged in this state, then I think it's best to set it to false in the useRef here.